_includes/news-section.html

```diff
@@ -0,0 +1,11 @@
+<article class="c-latest-news">
+<div class="c-latest-news__title-container">
+  <h1 class="c-latest-news__title">Latest News</h1>
+  <a href="#" class="c-latest-news__link c-latest-news__link--border">View All News</a>
+</div>
+<ul class="c-latest-news__link-list">
+  <li class="c-latest-news__list-item"><a href="#" class="c-latest-news__link">Dion Flannery Honored as Dayton Business Journal’s Executive of the Year Finalist</a><p class="c-latest-news__date">february 2017</p></li>
```

Comment:
Maybe for easier reading you could break these parts onto different lines, so it'd look something like this:

```html
<li>
  <a>...</a>
  <p>...</p>
</li>
```

Reply:
true 💯